firmware/common/spi_image.hpp

```diff
@@ -45,6 +46,14 @@ struct image_tag_t {
 	{
 	}
 
+	image_tag_t(const image_tag_t& other)
+	{
+		c[0] = other.c[0];
```

Comment:
Check the link in the commit message for more informations. C++ with -Wall is reporting a warning if you do not provide a constructor while providing operators

Reply:
Thanks to SiegeLord from allegro.cc's forum for helping me about that one. At first I didn't see the problem, because it's not a real problem: it's standardisation evolution that's causing the warning.

Reply:
Talking about that warning I think I remember seeing the same warning about some classes in looking glass (I or someone else may check)